lib/common.js

```diff
@@ -75,6 +108,7 @@ var parseSetEnumTypeDef = function(type){
   }else if(type.substr(0,5).toLowerCase() === 'enum('){
     prefixLen = 5;
   }else{
+    // TODO: Remove throw statement to make module safe
```

Comment:
This one should never happen... but it could also be emitted like the "Unsupported type..." error below.